src/components/stepforms/ConditionsEditor/ConditionsGroup.vue

```diff
@@ -147,17 +139,21 @@ export default class ConditionsGroup extends Vue {
     const newConditionsTree = {
       ...this.conditionsTree,
       groups: newGroups,
-    };
-    this.emitUpdatedConditionTree(newConditionsTree);
+    } as AbstractFilterTree;
+    // When there is only one condition, the operator is an empty string.
+    // Set the operator to 'and' when a group is added.
```

Comment:
This comment can be removed as it is inside `setOperatorIfNecessaryAndUpdateConditionTree`